cmd/destroy/remote.go

```diff
@@ -50,6 +50,7 @@ ENV {{ .NamespaceEnvVar }} {{ .NamespaceValue }}
 ENV {{ .ContextEnvVar }} {{ .ContextValue }}
 ENV {{ .TokenEnvVar }} {{ .TokenValue }}
 ENV {{ .RemoteDeployEnvVar }} true
+ENV {{ .ActionNameEnvVar }} {{ .ActionNameValue }}
```

Comment:
Can we add a test covering that we are adding or not this value?

Reply:
done a840a698ababb23144caa9d54e0f890d7e74a9b3